src/test/utils.ts

```diff
@@ -30,14 +30,16 @@ export const runQuery = (
     xForwardedFor: "123.456.789",
   }
 
-  return graphql(schema, query, rootValue, context).then(result => {
-    if (result.errors) {
-      const error = result.errors[0]
-      throw error.originalError || error
-    } else {
-      return result.data
-    }
-  })
+  return graphql(schema, query, rootValue, context)
+    .then(result => {
+      if (result.errors) {
+        const error = result.errors[0]
+        throw error.originalError || error
+      } else {
+        return result.data
+      }
+    })
+    .catch(error => console.error(error))
```

Comment:
With `catch` it points to the location, as well as provides a code frame directly to the test that triggered the error: 

<img width="634" alt="screen shot 2018-11-28 at 4 32 17 pm" src="https://user-images.githubusercontent.com/236943/49191748-aceb1c00-f32c-11e8-97bd-588280173ced.png">

Reply:
That said, the above isn't truly that useful since if the error originates outside of the test in, say, a schema file, the code-frame will point to the test location that triggered the error (failing test) but not the source-code where it originates. Pre-stitching I think the locations were a bit more clear.